tests/test_pixeldataset.py

```diff
@@ -70,6 +71,22 @@ def test_pixel_file_parquet_format_spec_can_save(setup_basic_pixel_dataset, tmp_
     assert file_target.is_file()
 
 
+def test_pixel_file_parquet_no_index_in_parquet_files(tmp_path):
+    # Checking EXE-1184
+    # We do not want an index added to the parquet file, regardless of if they
+    # read with pandas or polars
+    file_target = tmp_path / "df.parquet"
+    df = pd.DataFrame({"A": [1, 2, 3], "B": [1, 2, 3]}, index=["X", "Y", "Z"])
+    PixelFileParquetFormatSpec().serialize_dataframe(df, path=file_target)
+    assert file_target.is_file()
+    res1 = pd.read_parquet(file_target)
+    assert set(res1.columns) == {"A", "B"}
+    res2 = pl.scan_parquet(file_target)
+    assert set(res2.columns) == {"A", "B"}
+    res3 = pl.read_parquet(file_target)
```

Comment:
WHat are we testing here that is different to line 82?

Reply:
One reads with polars and one with pandas (`pl` vs. `pd`).

Reply:
Ah ya!